CMakeLists.txt

```diff
@@ -252,7 +257,7 @@ elseif(WIN32)
 
     # Configure file with custom definitions for NSIS.
     configure_file(
-        ${CMAKE_MODULE_PATH}/NSIS.definitions.nsh.in
+        cmake/NSIS.definitions.nsh.in
```

Comment:
why change this? this variable is set in this file and sets the absolute location

Reply:
it most definitely should not be a comma separated list, we set it like this `SET(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)` and we use it in other places as well

Reply:
I wonder why I was having issues with this on Windows. I'll switch it back and see if the build still works.